### PR TITLE
Update project test angular projects to conform to Angular v17 standards

### PR DESCRIPTION
**Pull request by hannahiss**
_Note: Please transform `- [ ]` into `- (NA)` in the description when things are not applicable_

### Related issues

Related to #100 

### Description

- For both projects angular-echarts and angular-ngx-echarts:
  - Make components standalone (now by default in v17)
  - Update app and routes configuration (more simple)
  - Update angular.json (deprecated browserTarget)
  - Remove unused nz-button and nzType Ant design attributes
- For project angular-tour-of-heroes/angular-echarts: 
  - Rename project angular-tour-of-heroes to angular-echarts
  - Add new example: same example than in project angular-ngx-echarts to compare both ways to use ods-charts in Angular (with or without ngx-echarts)
  - Adapt code to make it work without ngx-echarts
  - Make component line-chart more standard (extract html and scss, naming, etc...)

### Motivation & Context

- Differences on Angular config have been seen while creating angular-boosted-starter with ods-charts
- Make it easier to compare both ways to use ods-charts in Angular

### Types of change

- New feature (non-breaking change which adds functionality)
- Refactoring (non-breaking change)

### Test checklist

Please check that the following tests projects are still working:

- [ ] `docs/examples`
- [x] `test/angular-ngx-echarts`
- [x] `test/angular-echarts`
- [ ] `test/html`
- [ ] `test/react`
- [ ] `test/vue`
- [ ] `test/examples/bar-line-chart`
- [ ] `test/examples/single-line-chart`
- [ ] `test/examples/timeseries-chart`

### CODE REVIEW

**Review by julien-deramond — APPROVED** [review]
Thanks for the PR @hannahiss 
It was in draft but I've reviewed it anyway, hope you don't mind :)
I've made some minor modifications in https://github.com/Orange-OpenSource/ods-charts/pull/102/commits/51e412523fccbe44b5c1c711cd54a12519636c9f:
- Changed the README files to update the Angular versions. I think in the future for major versions, I'll rebuild the projects from scratch based on the `ng new` command
- Modified the formatting used by Angular for the imports: extra spaces around the names
- Changed `styleUrls` to `styleUrl` like it's done by default in their basic projects

Everything's working as before, so LGTM.

Do you have other modifications to do or can I merge it?

File: test/angular-ngx-echarts/README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
-# Angular
+# AngularTourOfHeroes
```
Comment (julien-deramond) [inline test/angular-ngx-echarts/README.md:1]:
```suggestion
# Angular
```

**Review by julien-deramond — CHANGES_REQUESTED** [review]
Oups, haven't spotted it in the first place but you've added a new chart for `AngularTourOfHeroes` example. No problem with that. However, when you click on a legend item, it doesn't hide the corresponding data:

**Comment by hannahiss** [comment]
> Oups, haven't spotted it in the first place but you've added a new chart for `AngularTourOfHeroes` example. No problem with that. However, when you click on a legend item, it doesn't hide the corresponding data:
> 

Yes, with LM we tried to make the same example than in angular-ngx-echarts to see the differences on how to use ods-charts with or without ngx-echarts. It is not totally finished, we wanted to talk to you about this... And we didn't have in mind this functionality... I'll check how to do it...

**Pull request merged**